### ISSUE TITLE
LittleBigPlanet Graphical Glitch - Lines appearing on textures

### ISSUE THREAD

**Issue by IndustryStandrd** [RPCS3/rpcs3#14761]
### Quick summary

![texture](https://github.com/RPCS3/rpcs3/assets/127009462/8382ca28-2117-4f5c-b9dc-2a03907668c7)

In LittleBigPlanet Game of the Year Edition all the materials have this grid line pattern over them. It's a lot more obvious and distracting when the camera is moving though. 

### Details

RPCS3 0.0.29-15694-23cb67e0 Alpha
Nvidia 3060 Ti 
Intel i5-13500
Windows 11 64bit
Little Big Planet 1 Game of the Year Edition
32GB Memory

Tried OpenGL mode, tried enabling strict rendering mode, disabling any upscaling, disabled/enabled antialiasing and AF

Tried looking on here and no ones mentioned these texture issues, just the known issue with transparency effects and the pod looking weird. Tried asking for support on Discord and no one seemed familiar so I don't think it has been reported yet. 

### Attach a log file

[RPCS3 (2).zip](https://github.com/RPCS3/rpcs3/files/13065129/RPCS3.2.zip)

### Attach capture files for visual issues

[BCUS98208_20231022211101_capture.zip](https://github.com/RPCS3/rpcs3/files/13065146/BCUS98208_20231022211101_capture.zip)

### System configuration

Nvidia 3060 Ti Driver 31.0.15.3734
Intel i5-113500 
32GB DDR5 Memory
Windows 11 Pro 64bit
Motherboard Gigabyte Z790 F7

**Comment by IndustryStandrd** [comment]
Realized my Nvidia driver was from August, got the latest October release and rebooted PC. Same graphical oddity.

**Comment by IndustryStandrd** [comment]
Going to point out this is happening on LBP2 as well.

**Comment by IndustryStandrd** [comment]
Happens for all 3 LittleBigPlanet games. 

Tried the emulator today on my Steamdeck with the same settings as PC (minus lower resolution) and it doesn't experience these gridlines on stickers.

**Comment by IndustryStandrd** [comment]
Completely wiped the install on my desktop and it was good for 2 levels then it the grid issue came back.

Turned on strict rendering and it stopped, managed to narrow down the problem and AF not being set to auto causes the gridlines. Not sure why this didn't work on the previous install though.

**Comment by TheGlitcher1705** [comment]
Setting anisotropic filtering to auto fixes this, I've added this to the RPCS3 wiki page for lbp1 but it also applies to lbp2

**Comment by TheWonderfulTartiflette** [comment]
Yep, happens to me and my friend as well, really distracting, would gladly need some help for this

**Comment by pollithor** [comment]
I have the same issue, did you find any solution?

**Comment by pollithor** [comment]
> Completely wiped the install on my desktop and it was good for 2 levels then it the grid issue came back.
> 
> Turned on strict rendering and it stopped, managed to narrow down the problem and AF not being set to auto causes the gridlines. Not sure why this didn't work on the previous install though.

It did work for me. Thank youuuu!

**Comment by digant73** [comment]
is that still an issue?

**Comment by kd-11** [comment]
Issue is caused by using AF setting that conflicts with the game's shaders.

**Comment by digant73** [comment]
Furthermore, wiki page is properly updated reporting `Anisotropic Filter` settings in `GPU` tab to be set to `Auto` to avoid this issue
